# The incoming command message from the coordinator

uint64 sendingTime #The sending time

uint16 messageTypeID

string message
